### PR TITLE
[F004] Calculate developer salary

### PR DESCRIPTION
**Pull request by ghost**
# Summary:
Calculate salary for NhanVienLapTrinh.
# Changes:
#### Change in Class NhanVienLapTrinh:
- [x] Change method toString() 
- [x] Add method get realSalary() 
# Checklist:
 - [x] Calculate salary for NhanVienKinhDoanh.
 - [x] Edit output to clear information.
 - [x] Reformat code with dart format.
# Result
Result

### CODE REVIEW

**Review by VuNA2k2 — APPROVED** [review]

**Review by Tris1702 — CHANGES_REQUESTED** [review]
Please add Description

File: NhanVienLapTrinh.dart [hunk 1]
```diff
@@ -1,32 +1,35 @@
 import 'NhanVien.dart';
 
-class NhanVienLapTrinh extends NhanVien {
+class NhanVienLapTrinh extends NhanVien{
```
Comment (Tris1702) [inline NhanVienLapTrinh.dart]: Revert

File: NhanVienLapTrinh.dart [hunk 2]
```diff
@@ -1,32 +1,35 @@
...
-      required String diaChi,
-      required double luong,
-      required this.level})
-      : super(
-            hoTen: hoTen,
-            ngaySinh: ngaySinh,
-            soDT: soDT,
-            diaChi: diaChi,
-            luong: luong) {
+  NhanVienLapTrinh({required String hoTen, required String ngaySinh, required String soDT, required String diaChi, required double luong, required this.level})
```
Comment (Tris1702) [inline NhanVienLapTrinh.dart]: Revert

**Review by Tris1702 — CHANGES_REQUESTED** [review]

File: NhanVienLapTrinh.dart (hunk 1 above)
Comment (ghost) [reply NhanVienLapTrinh.dart]: I reverted.

File: NhanVienLapTrinh.dart (hunk 2 above)
Comment (ghost) [reply NhanVienLapTrinh.dart]: I reverted

**Pull request merged**